src/firefly/js/visualize/ui/FileUploadViewPanel.jsx

```diff
@@ -545,7 +553,7 @@ function AnalysisInfo({report,supported=true,UNKNOWN_FORMAT}) {
 
 const tblOptions = {showToolbar:false, border:false, showOptionButton:false, showFilters:true};
 
-function AnalysisTable({summaryModel, detailsModel, report, isMoc, UNKNOWN_FORMAT, acceptList, acceptOneItem}) {
+function AnalysisTable({summaryModel, detailsModel, report, isMoc, UNKNOWN_FORMAT, acceptList, acceptOneItem, jobUrl, isUWS}) {
```

Comment:
I would think `jobUrl` would be in either one of these objects (summaryModel, detailsModel, report) already.  Do we need to pass specific variables into generic component?